frontend/packages/dev-console/src/components/import/git/GitSection.tsx

```diff
@@ -26,43 +31,55 @@ const GitSection: React.FC = () => {
   };
 
   return (
-    <FormSection title="Git" divider>
-      <InputField
-        type="text"
-        name="git.url"
-        label="Git Repo URL"
-        onBlur={handleGitUrlBlur}
-        required
-      />
-      {values.git.showGitType && (
-        <DropdownField
-          name="git.type"
-          label="Git Type"
-          items={GitTypes}
-          selectedKey={values.git.type}
-          title={GitTypes[values.git.type]}
-          fullWidth
-          required
-        />
+    <React.Fragment>
+      {values.git.secret.isNewSecret && (
+        <Alert
+          style={{ left: '800px', position: 'absolute', top: '20px', fontWeight: 'bold' }}
+          type="success"
+        >
+          Secret {values.git.secret.selectedKey} is created and linked with service account builder.
+        </Alert>
```

Comment:
@divyanshiGupta  Console has used `Alert` component of both pattenfly-3 and patternfly-4. Our Git Import form uses patternfly-3. To keep it consistent I have used patternfly-3's `Alert` component.

Reply:
We're transitioning to PF4 alerts now (#1763). We should use consistent alert styles across console... We should try to align on PF4.

Reply:
@debsmita1 why do you want to use alert instead of toasts? If you will use toasts you will not have to worry about adding any extra CSS. I will also suggest using PF4 toasts now as we are eventually going to move the forms to PF4 so we can start from here. Also, see if you can find any usage of toasts in the console and update it to use the PF4 component.

Reply:
So I just confirmed that toasts are not available in PF4, you can then use PF3 toast here. 
cc: @christianvogt

Reply:
@divyanshiGupta We don't use toasts anywhere else in console. I'd suggest we discuss toast notifications in general before adding them in just one place. I'd like to be consistent across console. We also had some negative feedback about our use of toasts in 3.x.

Reply:
We're also working hard to transition to PF4. Introducing a new PF3 component seems like a step back.

Reply:
@spadgett yes that makes sense, we need to discuss this with UX team since they have toast notifications in the design.